### PR TITLE
mtu workflows

### PR DESCRIPTION
**Pull request by anuragthehatter**
Opened in liu of https://github.com/openshift/release/pull/32541. There were lot of conflicts so safe to open this one. Addressed all comments here @lihongan @liangxia @zhaozhanqi 

Set cron values
Moved flows in respective aws and gcp secrtions

### CODE REVIEW

**Comment by liangxia** [comment]
There are conflicts.

**Comment by anuragthehatter** [comment]
@bandrade I probably need your merge approval on this while China is out on holidays.

**Comment by anuragthehatter** [comment]
@bandrade @sergiordlr Please help merging it. We can avoid cucushift failures as per https://github.com/openshift/release/pull/32541#issuecomment-1259045620. Thanks

**Comment by sergiordlr** [comment]
@anuragthehatter @lihongan  Please, the lgtm label  was removed due to recent changes. As soon as your team adds the lgtm label to the PR I'll approve it.

**Comment by sergiordlr** [comment]
@anuragthehatter sorry, I realized that I'm not in the list of approvers in this repo. It seems that in order to approve your PR you need to contact @dptp or @jupierce .

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -871,6 +871,16 @@ tests:
...
+- as: e2e-gcp-ipi-ovn-mtu-migrate-p2
+  cron: 0 22 */7 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      E2E_RUN_TAGS: '@gcp-ipi and @network-ovnkubernetes and not @fips'
+      TAG_VERSION: '@4.12'
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-gcp-ipi-ovn-mtu-migrate
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:883]: Extra white spaces at the end of this line cause `ci/prow/ci-operator-config-metadata`  failure.

Comment (anuragthehatter) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml]: Thanks @liangxia fixed it. That was bad :( Still trying to fix editor settings on my new refreshed laptop

**Pull request merged**